dateman.py

```diff
@@ -24,7 +24,11 @@ def __init__(self, date_to_key: str):
         self.url_base = DATE_URL
         self.holidays_frame = None
         self.last_updated ="2023-01-01"
-
+    def get_current_year(self):
```

Comment:
I'm not sure these are necessary. Isn't date time.today() enough?

Reply:
Done